src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -52,6 +53,27 @@
 }
 
 
+def configure_dynatrace_exporter(
+        endpoint_url: Optional[str] = None,
+        api_token: Optional[str] = None,
+        prefix: Optional[str] = None,
+        default_dimensions: Optional[Mapping[str, str]] = None,
+        export_dynatrace_metadata: Optional[bool] = False,
+        export_interval_millis: Optional[float] = None
```

Comment:
Would it make sense to set the defaults here instead of further down in the code? This way, users of this method can just hover over the method and see what they will get if they don't specify anything.